src/sssom/util.py

```diff
@@ -1044,46 +1044,41 @@ def get_dict_from_mapping(map_obj: Union[Any, Dict[Any, Any], SSSOM_Mapping]) ->
     :return: Dictionary
     """
     map_dict = {}
-    slots_with_double_as_range = [
-        s
-        for s in _get_sssom_schema_object().dict["slots"].keys()
-        if _get_sssom_schema_object().dict["slots"][s]["range"] == "double"
-    ]
+    sssom_schema_object = _get_sssom_schema_object().dict
+    slots = sssom_schema_object["slots"]
+    enums_keys = set(sssom_schema_object["enums"].keys())
+
+    slots_with_double_as_range = {k for k, v in slots.items() if v["range"] == "double"}
+
     for property in map_obj:
-        if map_obj[property] is not None:
```

Comment:
this diff is massive and impossible to understand, can you please make sure there is 100% test coverage on it

Reply:
Will do.

Reply:
Done.